scripts/send_message.rb

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env ruby
 
+# You probably always want this, right? Opens emails in browser
+ENV['LETTER_OPENER'] = '1'
```

Comment:
I thought about this and was worried it'd cause files to pop in demo or prod but then I realized that actually we only even bother to check this envvar in dev which means in other environments it will send 'live' emails.